src/tools/chip-cert/CertUtils.cpp

```diff
@@ -615,23 +618,42 @@ bool MakeCert(uint8_t certType, const ToolChipDN * subjectDN, X509 * caCert, EVP
         ReportOpenSSLErrorAndExit("X509_set_issuer_name", res = false);
     }
 
+    // Add basic constraints certificate extensions.
+    {
+        std::string basicConstraintsExt;
+
+        if (certType == kCertType_Node || certType == kCertType_FirmwareSigning)
```

Comment:
Suggest:

- Renaming `kCertType_Node` -> `kCertType_Leaf` since a DAC is not really a "node" certificate.
- Adding mention around `kCertType_FirmwareSigning` that it is for future use, since they are not used in Matter spec yet (other than being defined).

Reply:
Added additional description to the `kCertType_FirmwareSigning` declaration.